doc/src/Build_cmake.rst

```diff
@@ -22,27 +22,31 @@ Makefile(s). Example:
    cd lammps                        # change to the LAMMPS distribution directory
    mkdir build; cd build            # create a new directory (folder) for build
    cmake [options ...] ../cmake     # configuration with (command-line) cmake
-   make                             # compilation
+   make                             # compilation (or use "cmbuild" from "tools/cmake")
 
-The cmake command will detect available features, enable selected
+The ``cmake`` command will detect available features, enable selected
 packages and options, and will generate the build environment.  By default
 this build environment will be created for "Unix Makefiles" on most
 platforms and particularly on Linux.  However, alternate build tools
 (e.g. Ninja) and project files for Integrated Development Environments
 (IDEs) like Eclipse, CodeBlocks, or Kate can be generated, too. This is
-selected via the "-G" command line flag. For the rest of the documentation
+selected via the ``-G`` command line flag. For the rest of the documentation
 we will assume that the build environment is generated for makefiles
-and thus the make command will be used to compile and link LAMMPS as
-indicated above, producing (by default) an executable called "lmp" and
-a library called "liblammps.a" in the "build" folder.  When generating
+and thus the ``make`` command will be used to compile and link LAMMPS as
+indicated above, producing (by default) an executable called ``lmp`` and
+a library called ``liblammps.a`` in the ``build`` folder.  When generating
 a build environment for the "Ninja" build tool, the build command would
-be "ninja" instead of "make".
+be ``ninja`` instead of ``make``.  Or you may copy the ``cmbuild`` script
+from the :ref:`tools/cmake folder <cmake>` somewhere in your path and use that,
+as it will indirectly call CMake with the configured tool.  This is
+particularly, if you configured the build folder with a build tool
+in a non-standard location or name using ``-D CMAKE_MAKE_PROGRAM=<name>``.
 
 If your machine has multiple CPU cores (most do these days), using a
-command like "make -jN" (with N being the number of available local
+command like ``make -j N`` (with N being the number of available local
```

Comment:
It think we should say the `-j N` works on all the build tools: `make`, `ninja`, `cmake --build` and `cmbuild`.